go.mod

```diff
@@ -160,7 +160,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/pointer v0.59.0
 	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.59.0
 	github.com/DataDog/datadog-go/v5 v5.5.0
-	github.com/DataDog/datadog-operator v0.7.1-0.20241024104907-734366f3c0d1
+	github.com/DataDog/datadog-operator v0.7.1-0.20241111183642-43cd97e856a5
```

Comment:
temporary; this is referencing this commit https://github.com/DataDog/datadog-operator/pull/1519